FrontEnd/src/components/CheckBoxCustom.js

```diff
@@ -24,13 +24,22 @@ class CheckBoxCustom extends React.Component {
     );
   }
 
+  updateState () {
+    this.setState({
+        elements: Object.assign({}, this.state.elements[1], {
+            disabled: 'true'
+        })
+    });
+  }
+
```

Comment:
not quite sure why you need this / what this is doing
you're never calling it?

Reply:
forgot to remove it